ext/standard/streamsfuncs.c

```diff
@@ -970,8 +972,8 @@ PHP_FUNCTION(stream_context_get_options)
 
 	context = decode_context_param(zcontext);
 	if (!context) {
-		php_error_docref(NULL, E_WARNING, "Invalid stream/context parameter");
-		RETURN_FALSE;
+		zend_value_error("Invalid stream/context parameter");
```

Comment:
Possibly these should also be TypeError? These are basically the same as the "invalid resource type" errors, just that there's two possible types here.